### ISSUE TITLE
Version regex broken for non-single digits versions 

### ISSUE THREAD

**Issue by Raymondd** [Microsoft/vscode-extension-vscode#53]
I get 
```
Current working directory: W:\vscode-mssql
Downloading VS Code into "W:\vscode-mssql\.vscode-test" from: https://vscode-update.azurewebsites.net/1.10.0/win32-archive/stable
https://vscode-update.azurewebsites.net/1.10.0/win32-archive/stable
Error running tests: TypeError: Cannot read property '0' of null
```

After some debugging I found that the regex on line 82: 
`var version = downloadUrl.match(/\d\.\d\.\d/)[0].split('\.');`
does not catch double digit versions and the new version of VSCode is `1.10.0`.

**Comment by bpasero** [comment]
Awesome catch!

**Comment by bpasero** [comment]
I have published 1.0.5 with this fix.
